versions/3.0.md

```diff
@@ -2253,25 +2254,21 @@ $ref: 'Pet.yaml'
 ```
 
 ```yaml
-$ref: 'definitions.yaml#/Pet'
+$ref: definitions.yaml#/Pet
 ```
 
 #### <a name="schemaObject"></a>Schema Object
 
 The Schema Object allows the definition of input and output data types.
 These types can be objects, but also primitives and arrays.
-This object is based on the [JSON Schema Specification Draft 4](http://json-schema.org/) and uses a predefined subset of it.
-On top of this subset, there are extensions provided by this specification to allow for more complete documentation.
+This object is an extended subset of the [JSON Schema Specification Wright Draft 00](http://json-schema.org/).
 
-Further information about the properties can be found in [JSON Schema Core](http://json-schema.org/latest/json-schema-core.html) and [JSON Schema Validation](http://json-schema.org/latest/json-schema-validation.html).
+Further information about the properties can be found in [JSON Schema Core](https://tools.ietf.org/html/draft-wright-json-schema-00) and [JSON Schema Validation](https://tools.ietf.org/html/draft-wright-json-schema-validation-00).
```

Comment:
JSON Schema is split into two specifications - Core and Validation. Can't keep it as one, and shouldn't reference both every single time.